### PR TITLE
AMM-CDA-8 Add event information for fees and aggregated amount_out

### PR DESCRIPTION
**Pull request by Chralt98**

### CODE REVIEW

**Comment by Chralt98** [comment]
CI failed for a test. I fix it quickly.

File: primitives/src/traits/hybrid_router_amm_api.rs [hunk 1]
```diff
@@ -89,7 +93,7 @@ pub trait HybridRouterAmmApi {
         asset_out: Self::Asset,
```
[the remark this answers is not in the document]
Comment (Chralt98) [reply primitives/src/traits/hybrid_router_amm_api.rs:93]: ad0ec1c6244803fc5b5a3bf6ea38c1ed72ed5c42

File: primitives/src/traits/hybrid_router_amm_api.rs [hunk 2]
```diff
@@ -129,5 +133,5 @@ pub trait HybridRouterAmmApi {
         asset_in: Self::Asset,
         amount_in: Self::Balance,
```
[the remark this answers is not in the document]
Comment (Chralt98) [reply primitives/src/traits/hybrid_router_amm_api.rs:134]: ad0ec1c6244803fc5b5a3bf6ea38c1ed72ed5c42

File: zrml/hybrid-router/Cargo.toml [hunk 3]
```diff
@@ -30,6 +30,7 @@ zrml-neo-swaps = { workspace = true, optional = true }
 zrml-orderbook = { workspace = true, optional = true }
 zrml-prediction-markets = { workspace = true, optional = true }
 zrml-simple-disputes = { workspace = true, optional = true }
+zrml-swaps = { workspace = true, optional = true }
```
[the remark this answers is not in the document]
Comment (Chralt98) [reply zrml/hybrid-router/Cargo.toml]: 16669c7633a26ecc39df26840756f4b694a6c4c1

**Review by maltekliemann — APPROVED** [review]

**Pull request merged**